src/Services/Ordering/Ordering.Domain/AggregatesModel/DTOs/OrderItemDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Ordering.Domain.AggregatesModel.DTOs
+{
+    public class OrderItemDto
```

Comment:
Wouldn't be better to keep `OrderItemDto` file outside of `AggregatesModel` but within the `Ordering.Domain` project ?
That way the DTO stays close to the Order aggregate model but stays separate than AggregatesModel. Thoughts ?